dockerfiles/hpcc/templates/thor.yaml

```diff
@@ -1,84 +1,137 @@
 {{ range $thor := $.Values.thor -}}
 {{- $masterPort := $thor.masterport | default 20000 -}}
 {{- $slavePort := $thor.slaveport | default 20100 -}}
+{{- $eclagentName := printf "%s-agent" .name }}
 {{- $slaveName := printf "%s-slave" .name }}
 {{- $serviceName := printf "%s-svc" .name }}
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .name | quote }}
+  name: {{ $eclagentName | quote }}
 spec:
-  replicas: 1
+  replicas: {{ $thor.eclagent.replicas | default 1 }}
   selector:
     matchLabels:
-      run: {{ .name | quote }}
+      run: {{ $eclagentName | quote }}
   template:
     metadata:
       labels:
-        run: {{ .name | quote }}
+        run: {{ $eclagentName | quote }}
     spec:
-      {{- include "hpcc.checkDataStorageHostMount" (dict "root" $) | indent 6 }}
+      serviceAccountName: hpcc
       containers:
-      - name: {{ .name | quote }}
+      - name: {{ $eclagentName | quote }}
         ports:
           - containerPort: {{ $masterPort }}
         args: [
                 {{ include "hpcc.configArg" . }},
-                {{ include "hpcc.daliArg" $ }},
-                --masterport={{ $masterPort }},
-                --numSlaves={{ $thor.numSlaves }}
+                {{ include "hpcc.daliArg" $ }}
+                {{- if $thor.maxActive }},
```

Comment:
Shouldn't need things like this now we pass all settings through via config file

Reply:
right.. I think was under a different section at some point. will remove.